src/applications/personalization/profile/util/direct-deposit.js

```diff
@@ -0,0 +1,55 @@
+import cloneDeep from 'lodash/cloneDeep';
+import set from 'lodash/set';
+
+export class DirectDepositClient {
+  #PPIU_ENDPOINT = '/ppiu/payment_information';
+
+  #LH_ENDPOINT = '/profile/direct_deposits/disability_compensations';
+
+  constructor({ useLighthouseDirectDepositEndpoint = false }) {
+    this.useLighthouseEndpoint = useLighthouseDirectDepositEndpoint;
+  }
+
+  get endpoint() {
+    return this.useLighthouseEndpoint ? this.#LH_ENDPOINT : this.#PPIU_ENDPOINT;
+  }
+
+  generateApiRequestOptions(payload) {
+    const options = {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      method: 'PUT',
+      body: JSON.stringify(payload),
+      mode: 'cors',
+    };
+
+    if (this.useLighthouseEndpoint) {
+      options.body = JSON.stringify({
+        accountType: payload.accountType,
+        accountNumber: payload.accountNumber,
+        routingNumber: payload.financialInstitutionRoutingNumber,
+      });
+    }
+    return options;
+  }
+
+  formatDirectDepositResponseFromLighthouse = response => {
+    const result = cloneDeep(response);
+    if (result?.paymentAccount?.name) {
+      set(
+        result,
+        'paymentAccount.financialInstitutionName',
+        result.paymentAccount.name,
+      );
+    }
+    if (result.paymentAccount?.routingNumber) {
+      set(
+        result,
+        'paymentAccount.financialInstitutionRoutingNumber',
+        result.paymentAccount.routingNumber,
+      );
+    }
+    return result;
+  };
+}
```

Comment:
So this is the new 'client adapter' class. It basically provides some convenience items to get the api endpoint to be used, format request options, and format response data.

I'd like to expand upon this and maybe move more Direct Deposit based code into this class, but it works for now.

Reply:
I seem to recall from a Lighthouse meeting that they wanted to know who was using the ppiu endpoint from EVSS - I guess they know about this one?

Reply:
Yep, they are aware, and basically this is in preparation of phasing that `ppiu` endpoint usage out to use a lighthouse based endpoint.